tg_forwarder_client.py

```diff
@@ -28,12 +33,27 @@ def get_settings() -> Settings:
     return Settings()
 
 
-async def channel_handler(_, message):
+def send_media_group(client, media_group_id):
+    settings = get_settings()
+
+    for chat_id in settings.chats:
+        loop.create_task(client.forward_messages(chat_id, *queues[media_group_id]))
+
+
+async def channel_handler(client, msg):
     settings = get_settings()
-    if message.chat.username in settings.channels and not message.edit_date:
-        logging.info(message.chat)
-        for chat_id in settings.chats:
-            await message.forward(chat_id)
+    if msg.chat.username in settings.channels and not msg.edit_date:
```

Comment:
Надо всегда пытаться избавляться от избыточной вложенности

Например, так:

```suggestion
    if msg.chat.username not in settings.channels or msg.edit_date:
        return
```